leo/src/app/admin/events/eventForm.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+
+import React, { useEffect } from 'react';
+import { useFormState, useFormStatus } from 'react-dom';
+import { addEvent, FormState } from '@/actions/event';
+
+function SubmitButton() {
+    const { pending } = useFormStatus();
+
+    return (
+        <button disabled={pending} type="submit" className="bg-blue-500 text-white p-2 rounded">
```

Comment:
`Button`

Reply:
We cant just use the Button component without making some changes to it first. We would need to add some additional functionality to the buttons because we are currently using the disabled tag which is not supported on the Button Component